pkg/pipeline/PipelineBuilder.go

```diff
@@ -414,18 +414,21 @@ func (impl PipelineBuilderImpl) GetApp(appId int) (application *bean.CreateAppDT
 		impl.logger.Errorw("error in fetching app", "id", appId, "err", err)
 		return nil, err
 	}
-
+	application = &bean.CreateAppDTO{
+		Id:      app.Id,
+		AppName: app.AppName,
+		TeamId:  app.TeamId,
+		AppType: app.AppType,
```

Comment:
AppType should be DeploymentType or DeploymentAppType

Reply:
this object had usages in almost above 20 files, changing object would require affecting those files also, I can change the meaning or I can just add a comment here that it means deployment app type, also this dto was there before my code.
So should I change it to DeploymentAppType